src/routes/search/containers/__tests__/SearchPage.spec.js

```diff
@@ -175,7 +175,7 @@ describe('SearchPage', () => {
       <ConnectedSearchPage store={store} categories={categories} />
     )
 
-    expect(searchPage.props()).toMatchObject({
+    expect(searchPage.dive().dive().props()).toMatchObject({
```

Comment:
Why don't you use `.find()` as above?

Reply:
find() does not render deeper. If there is nothing rendered then find returns null.

Dive renders the dom one level deeper.